### DISCUSSION TITLE
Is there support for TraceQL via Tempo API in Uptrace v. 1.7.0?

### DISCUSSION THREAD

**Discussion by andrewkewang** [uptrace/uptrace#358] [Q&A]
The [changelog](https://github.com/uptrace/uptrace/blob/master/CHANGELOG.md#v170---unreleased) for v 1.7.0 mentions "Updated Tempo API to the latest version." 

Current Tempo API has a[ search API that supports TraceQL](https://grafana.com/docs/tempo/latest/api_docs/#search). Is this TraceQL search syntax supported via Uptrace?

**Comment by vmihailenco** [comment]
Uptrace only supports TraceQL filters/conditions: https://github.com/uptrace/uptrace/blob/master/pkg/grafana/tempo_handler.go#L380-L407 . Some intrinsic filters might not be supported.

Let me know what TraceQL features you're interested in and I will see if we can support them.
